apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -92,6 +95,9 @@ const AvailabilityPage = ({ profile, plan, eventType, workingHours, previousPage
   }
   const [isTimeOptionsOpen, setIsTimeOptionsOpen] = useState(false);
   const [timeFormat, setTimeFormat] = useState(detectBrowserTimeFormat);
+  const [recurringEventCount, setRecurringEventCount] = useState(
+    eventType.recurringEvent && eventType.recurringEvent.count
+  );
```

Comment:
Used to keep track of value changes to switch dependent UI elements, mostly translation plurals